specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2020-10-01/ScaleUnitNode.json

```diff
@@ -315,19 +315,19 @@
         "operationId": "ScaleUnitNodes_List",
         "parameters": [
           {
-            "$ref": "Fabric.json#/parameters/SubscriptionIdParameter"
+            "$ref": "../2016-05-01/Fabric.json#/parameters/SubscriptionIdParameter"
```

Comment:
is it recommended to make a copy of Fabric.json to 2020-10-01 folder ? @allenjzhang

Reply:
Direct copying this file, will cause a version problem. Other models are still in old version.  I just followed the existing solution of referencing the old model, like here:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2018-10-01/Drive.json